container.go

```diff
@@ -147,41 +148,67 @@ func showContainerCheckpoint(checkpointDirectory string) error {
 	table.Append(row)
 	table.Render()
 
-	if !printStats {
-		return nil
+	if showMounts {
+		table = tablewriter.NewWriter(os.Stdout)
+		table.SetHeader([]string{
+			"Destination",
+			"Type",
+			"Source",
+		})
+		// Get overview of mounts from spec.dump
+		for _, data := range specDump.Mounts {
+			table.Append([]string{
+				data.Destination,
+				data.Type,
+				func() string {
+					if fullPaths {
+						return data.Source
+					}
+					return shortenPath(data.Source)
+				}(),
+			})
+		}
+		fmt.Println("\nOverview of Mounts")
+		table.Render()
+	} else {
+		if fullPaths {
+			return fmt.Errorf("Cannot use --full-paths without --mounts option")
```

Comment:
Thanks. Please do this check much earlier. I would do it in `checkpointctl.go` before opening any files. No need to unpack a file to detect later that we will not use it at all.